crates/svm-runtime-c-api/src/api.rs

```diff
@@ -936,7 +917,7 @@ pub unsafe extern "C" fn svm_app_receipt_addr(
 ///
 #[no_mangle]
 pub unsafe extern "C" fn svm_app_receipt_returns(
-    returns: *mut svm_value_array,
+    returns: *mut svm_byte_array,
```

Comment:
I think we should document here and in other functions as well that `svm_byte_array` is the encoded format of `Vec<WasmValue`>/`Vec<WasmType>`.
Otherwise, it won't be clear how one should construct that field.